cli/command/context/list.go

```diff
@@ -111,6 +117,28 @@ func runList(dockerCli command.Cli, opts *listOptions) error {
 	return nil
 }
 
+// getContextType sets the LegacyContextType field for compatibility with
+// Visual Studio, which depends on this field from the "cloud integration"
+// wrapper.
+//
+// https://github.com/docker/compose-cli/blob/c156ce6da4c2b317174d42daf1b019efa87e9f92/api/context/store/contextmetadata.go#L28-L34
+// https://github.com/docker/compose-cli/blob/c156ce6da4c2b317174d42daf1b019efa87e9f92/api/context/store/store.go#L34-L51
+func getContextType(meta map[string]any, format string) string {
```

Comment:
just out of an abundance of caution, should we also signal that this shouldn't be used for anything aside of this patch? or directly leave a TODO to remove this one the special binary is no longer needed?

Reply:
Ah, yeah, I thought; let me mark the field as deprecated to be "somewhat" of a reminder; don't use. And this function is non-exported, so at least nobody would be able to use it.